### ISSUE TITLE
Backslash? Forward slash

### ISSUE THREAD

**Issue by flowsta** [ckan/ckanext-dcat#245]
The section [Catalog Endpoint](https://github.com/ckan/ckanext-dcat#catalog-endpoint) states that the *endpoint* can be customized:

```
ckanext.dcat.catalog_endpoint = /dcat/catalog/{_format}
```
The explanation says:

> The custom endpoint must start with a backslash (/) and contain the {_format} placeholder.

Backslash? Nop, backslash is `\`. You mean **forward slash**, `/`.

So...
```
sed 's/backslash/forward slash/'  README.md
```

**Comment by amercader** [comment]
Thanks @flowsta , do you mind submitting a PR with the changes needed?

**Comment by seitenbau-govdata** [comment]
Fixed by #246
